src/event-replay/event-replay.ts

```diff
@@ -112,11 +113,13 @@ export async function importEventsFromTsv(
   const blockWindowSize = parseInt(
     process.env['STACKS_MEMPOOL_TX_GARBAGE_COLLECTION_THRESHOLD'] ?? '256'
   );
-  const prunedBlockHeight = Math.max(tsvBlockHeight - blockWindowSize, 0);
+  const prunedBlockHeight =
+    prunedBlockHeightOption ?? Math.max(tsvBlockHeight - blockWindowSize, 0);
   console.log(`Event file's block height: ${tsvBlockHeight}`);
   console.log(`Starting event import and playback in ${eventImportMode} mode`);
   if (eventImportMode === EventImportMode.pruned) {
     console.log(`Ignoring all prunable events before block height: ${prunedBlockHeight}`);
+    process.env.IBD_MODE_UNTIL_BLOCK = `${prunedBlockHeight}`;
```

Comment:
Can the new `IBD_MODE_UNTIL_BLOCK` var be "documented" in the `.env` file? (quotes because the .env file is probably not the best place to document these but it's what we have today 😅 )

Just something like
```
# Uncomment to enable "initial block download mode", while syncing this will ....
# IBD_MODE_UNTIL_BLOCK=100
```

Reply:
Oops nvm! I missed where you had already done this! Thanks!